src/custom.py

```diff
@@ -58,11 +58,21 @@ async def define_cmd(name, response: str, author: discord.User | discord.Member)
     # Format confirmation to the user
     output_message = f"New command added! You can use it like `{CMD_PREFIX}{name}`. "
 
```

Comment:
Rather than do this handling, I think it's probably better to just always put the before and after into their own blocks, even if they're not very long. Something like

Foo has changed the command bar from
```
before
```
to
```
after
```